src/wing/CCPACSWing.h

```diff
@@ -64,125 +78,280 @@ friend class CTiglWingBuilder;
     TIGL_EXPORT void SetSymmetryAxis(const TiglSymmetryAxis& axis) override;
     TIGL_EXPORT void SetParentUID(const boost::optional<std::string>& value) override;
 
-    // Returns whether this wing is a rotor blade
+    /**
+     * @brief Returns whether this wing is a rotor blade
+     * @return bool
+     */
     TIGL_EXPORT bool IsRotorBlade() const;
 
-    // Returns the parent configuration
+    /**
+     *  @brief Returns the parent configuration
+     *  @return CCPACSConfiguration&
+     */
     TIGL_EXPORT CCPACSConfiguration & GetConfiguration() const;
 
-    // Get section count
+    /**
+     * @brief Get section count
+     * @return int
+     */
     TIGL_EXPORT int GetSectionCount() const;
 
-    // Returns the section for a given index
+    /**
+     * @brief Returns the section for a given index
+     * @return CCPACSWingSection&
+     */
     TIGL_EXPORT CCPACSWingSection& GetSection(int index);
     TIGL_EXPORT const CCPACSWingSection& GetSection(int index) const;
```

Comment:
I am not sure, if there will be documentation for the const version of the function. Could you please check the generated html? I am not sure what is better: The duplication of documentation or missing documentation.

Reply:
In general I don't think simple duplication is the right way to deal with overloaded functions, so I will look into each case. But I take it, from everything I know about documentation, that we should never be afraid to repeat ourselves. And I really don't think that repetition is irritating in documentation, I would even say that repetition is in many cases important for the user because one probably can't take that information from context.

Reply:
I agree with everything you say, but there is also the downside to repetition that we now have to maintain two docstrings: It is all to easy to change the docstring of one of the two functions and forgetting to update the other. 

Since typically overloads do different things we should have different documentations for the overloads. But here the difference is only in the const specifier on the method and return type, so the difference is already conveyed by the signature.

Reply:
ok, I understand that, but in general I suppose it is not gueranteed, even with similar functions like that, that they will be modified in the same way, and we will have to check that anyway, and while doing so, we would check each documentation. I think making an exception for a case like that would be even more confusing about what is to be documented and what should be left out. Also, the user might need to be sure about the behavior really being the same?

Reply:
I agree. Either no overload gets a docstring, or both.